amf-cli/shared/src/test/resources/upanddown/oas31/oas-31-full.yaml

```diff
@@ -69,7 +73,7 @@ components:
         email:
           type: string
     OtherSchema:
-      $schema: https://spec.openapis.org/oas/3.1/dialect/base
+      # $schema: https://spec.openapis.org/oas/3.1/dialect/base # TODO: still not implemented
```

Comment:
Why comment this? Is not supported but it should be parsed and emitted. The test is OK